src/main/native/compression/IntelDeflater.cc

```diff
@@ -37,7 +37,10 @@ extern "C" {
 #include "igzip_lib.h"
 }
 
+static struct isal_huff_histogram histogram;
```

Comment:
Making `histogram` a global variable means we only have one `histogram` per Java application. I think this is a problem.